forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -151,8 +153,14 @@ internal class CapturePaymentRepository(
                 return ForageApiResponse.Failure(listOf(ForageError(500, "unknown_server_error", "Unknown Server Error")))
             }
 
+            var intervalTime: Long = if (pollingIntervals != null && attempt < pollingIntervals.size) {
+                pollingIntervals[attempt].toLong()
+            } else {
+                1000L
```

Comment:
Ah I see you default to polling for every second if there are issues with the `pollingIntervals` value.

Reply:
yes, we'll fall back to 1 second if `pollingIntervals` was not populated correctly or if the length of the array is less than the number of max attempts.

If length of pollingIntervals is 8, but max attempts is 10, the last 2 attempts would wait 1 second each before retrying